modules/log.js

```diff
@@ -53,16 +53,23 @@ export function log() {
         callback(null, file);
       }))
       .on('finish', () => {
-        let t = 1;
```

Comment:
Looks like you removed the numbering for the errors. Let's keep those in please.